package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "sunrise",
+  "version": "1.0.0",
+  "description": "[![Build Status](https://travis-ci.org/commercetools/commercetools-sunrise-data.svg?branch=master)](https://travis-ci.org/commercetools/commercetools-sunrise-data)",
+  "main": "index.js",
+  "scripts": {
+    "get:config": "echo \"authUrl\": && npm config get commercetools-sunrise-data:authUrl && echo \"apiUrl\": && npm config get commercetools-sunrise-data:apiUrl && echo \"projectKey\": && npm config get commercetools-sunrise-data:project && echo \"clientId\": && npm config get commercetools-sunrise-data:client_id && echo \"clientSecret\": && npm config get commercetools-sunrise-data:client_secret",
+    "build": "rimraf dist/ && babel ./**/*.js --out-dir dist/ --ignore ./node_modules,./.babelrc,./package.json,./npm-debug.log --copy-files",
+    "precredentials": "npm run build",
+    "credentials": "node dist/lib/update-credentials.js --project $npm_package_config_project --client_id $npm_package_config_client_id --client_secret $npm_package_config_client_secret --apiUrl $npm_package_config_apiUrl --authUrl $npm_package_config_authUrl",
+    "clean:data": "./mvnw spring-boot:run -DprojectCleanUp=true -DprojectSetUp=false -DchannelsImport=false -DcatalogImport=false -DreserveInStore=false -DordersImport=false -Dctp.projectKey=$npm_package_config_project -Dctp.clientId=$npm_package_config_client_id -Dctp.clientSecret=$npm_package_config_client_secret -Dctp.authUrl=$npm_package_config_authUrl -Dctp.apiUrl=$npm_package_config_apiUrl",
```

Comment:
Regression:  you have renamed the conf variables (camel case projectKey now) but have not completely adjusted the places where they are used. 

Could you run a complete test roundrip please before our next iteration?   (i.e. fresh into a new project and then again when the project already contains data)

```
Error creating bean with name 'sphereClient' defined in class path resource [com/commercetools/dataimport/SphereClientConfiguration.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [io.sphere.sdk.client.BlockingSphereClient]: Factory method 'sphereClient' threw exception; nested exception is java.lang.IllegalArgumentException: projectKey is null or empty.
```

Reply:
tried to do it quickly just before my call 👎 
But it turns out to be a good thing, as there was inconsistency in the name `project` and `projectKey`. renamed all to `projectKey`